src/aura/HH_ContactCard/HH_ContactCard.cmp

```diff
@@ -9,13 +9,13 @@
     <!-- markup follows... -->
 
     <div class="slds-card slds-p-around--none">
-        <div class="slds-card__header slds-p-around--none">
+        <div class="slds-card__header slds-p-around--small">
```

Comment:
did you do any side by side testing to make sure the end results was the same, or do I need to test this out in my own org to verify?  thanks.